coalesce/organizations/test/test_views.py

```diff
@@ -0,0 +1,24 @@
+from django.urls import reverse
+from nose.tools import eq_
+from rest_framework.test import APITestCase
+from rest_framework import status
+from ..models import Organization
+from ...organizations.test.factories import OrganizationFactory
+
+
+class TestOrganizationCreate(APITestCase):
+    """
+    Tests /organizers create operations.
+    """
+
+    def setUp(self):
+        self.organization = OrganizationFactory()
+        self.client.credentials(HTTP_AUTHORIZATION=f'Token {self.user.auth_token}')

```

Comment:
This test does not pass with `docker-compose exec api ./manage.py test`:
```AttributeError: 'TestOrganizationCreate' object has no attribute 'user```

Adding a User should solve that:
```python
self.user = UserFactory()```

The `test_post_request_with_valid_data_succeeds()` test fails because HTTP 400 Bad Request is returned instead of 201. I think something is still missing to get the test passing.

Reply:
This should be fixed now